homeassistant/components/sensor/teamspeak.py

```diff
@@ -0,0 +1,128 @@
+"""
+homeassistant.components.sensor.teamspeak
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Monitors a teamspeak server
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.teamspeak/
+"""
+import logging
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['ts3==0.7.1']
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = 'Teamspeak Server'
+ATTR_SERVER_NAME = 'Name'
+ATTR_MAX_USERS = 'Max. Users'
```

Comment:
Thank you, I try to look at other code for examples and find several other modules not respecting this then:
-cpuspeed
-swiss_public_transport

I cannot find a single sensor that does it another way , except maybe vera, they don't use a variable but just put it in the code

`attr['Vera Device Id'] = self.vera_device.vera_device_id`

Is that better? Looks strange?
Maybe just remove the attributes? Showing something like "max_users" in frontend doesn't seem too user friendly?